sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetTest.scala

```diff
@@ -63,13 +63,38 @@ private[sql] trait ParquetTest extends FileBasedDataSourceTest {
       (f: String => Unit): Unit = withDataSourceFile(data)(f)
 
   /**
-   * Writes `data` to a Parquet file and reads it back as a [[DataFrame]],
+   * Writes `data` objects to a Parquet file and reads it back as a [[DataFrame]],
    * which is then passed to `f`. The Parquet file will be deleted after `f` returns.
    */
-  protected def withParquetDataFrame[T <: Product: ClassTag: TypeTag]
+  protected def withParquetDFfromObjs[T <: Product: ClassTag: TypeTag]
```

Comment:
Can we have this renaming in a separate `TEST` PR first?

Reply:
Sounds good idea.